File: android/versioned-abis/expoview-abi42_0_0/src/main/java/abi42_0_0/expo/modules/font/FontLoaderPackage.java
package abi42_0_0.expo.modules.font;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi42_0_0.org.unimodules.core.BasePackage;
import abi42_0_0.org.unimodules.core.ExportedModule;

public class FontLoaderPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new FontLoaderModule(context));
  }
}

File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/font/FontLoaderPackage.kt
package abi43_0_0.expo.modules.font

import android.content.Context
import abi43_0_0.expo.modules.core.BasePackage
import abi43_0_0.expo.modules.core.ExportedModule

class FontLoaderPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf<ExportedModule>(FontLoaderModule(context))
  }
}
